### PR TITLE
DanielSherratt Just-Install Updates

### PR DESCRIPTION
**Pull request by danielsherratt**

### CODE REVIEW

File: just-install-v4.json [hunk 1]
```diff
@@ -165,12 +165,12 @@
       },
       "version": "latest"
     },
-    "audacity": {
+	"audacity": {
```
Comment (mimi89999) [inline just-install-v4.json:168]: Please fix indent

**Review by mimi89999 — CHANGES_REQUESTED** [review]
Please fix indent

**Comment by mimi89999** [comment]
Please rebase on master

**Comment by danielsherratt** [comment]
Using winget

**Pull request closed without merging**